Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```diff
@@ -72,38 +72,44 @@ configuration:
   name: credentials
   type: 9
   required: false
+- additionalinfo: The API Root to use (e.g. default, public). If left empty, the server default API root will be used. When the server has no default root, the first available API root will be used instead.
```

Comment:
Recommend:
The API root to use (for example default or public). If left empty, the server default API root is used. If the server has no default root, the first available API root is used instead.